cpuinfo.py

```diff
@@ -941,7 +941,7 @@ def get_cpu_info_from_cpuid():
 	'''
 
 	returncode, output = run_and_get_stdout([sys.executable, "-c", "import cpuinfo; print(cpuinfo.actual_get_cpu_info_from_cpuid())"])
-	if returncode != 0:
+	if (returncode != 0) or (output.strip() == 'None'):
 		return None
```

Comment:
The standard indentation size in Python is 4 spaces

Reply:
I'm pretty confused by your request. Your `cpuinfo.py` uses tabs, not spaces for indentation, and as little as I like it, I kept with the existing style. The line in question is indented with a single tab character, as the original line was before I made my edits.

```
$ head -n 944 cpuinfo.py|tail -n 1 | xxd
00000000: 0969 6620 2872 6574 7572 6e63 6f64 6520  .if (returncode
00000010: 213d 2030 2920 6f72 2028 6f75 7470 7574  != 0) or (output
00000020: 2e73 7472 6970 2829 203d 3d20 274e 6f6e  .strip() == 'Non
00000030: 6527 293a 0a                             e'):.
```

Reply:
`autopep8 -i cpuinfo.py` completed successfully, I'm happy to push the resulting tab-free file if you'd like.

Reply:
Oh, I was not aware that `cpuinfo.py` was using tabs instead of spaces.  Never mind, in order to not change too much, I'll merge as is.  Thanks!